src/FileUpload/File.php

```diff
@@ -22,15 +22,21 @@ class File extends \SplFileInfo
      */
     protected $mimeType = 'application/octet-stream';
 
+    /**
+     * @var mixed
+     */
+    protected $clientFileName;
+
     /**
      * Is the file completely downloaded
      * @var boolean
      */
     public $completed = false;
 
-    public function __construct($fileName)
+    public function __construct($fileName, $clientFileName=false)
```

Comment:
Can this be an empty string instead ? like `public function __construct($fileName, $clientFileName)`... I don't think it would nice to have `getClientFileName()` return false .

Again, is there any reason why it has a default, it should default to an empty string anyways ( or so I think )

Reply:
I made the change like this because I saw 3 options: touch the constructor, to add both public getter and setter or do only public property with the file name.. if you are concerned about the default value in constructor, we cannot remove it without breaking compatibility and is better to rely on the others ways I wrote before. Btw sure I can make the property string empty by default... Let me know which option better fits for you.

Reply:
I think making the `$clientFileName` empty by default works